src/github/copilotRemoteAgent.ts

```diff
@@ -791,11 +791,14 @@ export class CopilotRemoteAgentManager extends Disposable {
 		const status = copilotEventToSessionStatus(mostRecentCopilotEvent(timeline));
 		const tooltip = await issueMarkdown(pullRequest, this.context, this.repositoriesManager);
 		const timestampNumber = new Date(pullRequest.createdAt).getTime();
+		const defaultBranch = await pullRequest.githubRepository.getDefaultBranch();
+		const description = pullRequest.base.ref === defaultBranch ? `PR #${pullRequest.number}` : `PR #${pullRequest.number} → ${pullRequest.base.ref}`;
```

Comment:
We have mostly stayed away from abreviating PR. I see a few have slipped past me over the years though. I'll make a pass through and switch them all over to "pull request".